rrbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  
  # Position Controllers ---------------------------------------

  verin_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint1
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
  verin_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint2
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
  verin_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint3
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
  verin_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint4
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
  verin_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint5
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
  verin_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: verin_joint6
    # pid: {p: 100.0, i: 1.0, d: 0.0}
    pid: {p: 10.0, i: 0.01, d: 0.01}
